pycti/utils/opencti_stix2.py

```diff
@@ -1623,31 +1624,38 @@ def prepare_export(
         )
         for stix_nested_ref_relationship in stix_nested_ref_relationships:
             if "standard_id" in stix_nested_ref_relationship["to"]:
-                if MultipleRefRelationship.has_value(
+                # dirty fix because the sample and operating-system ref are not multiple for a Malware Analysis
+                # will be replaced by a proper toStix converter in the back
+                if not MultipleRefRelationship.has_value(
                     stix_nested_ref_relationship["relationship_type"]
+                ) or (
+                    entity["type"] == "malware-analysis"
+                    and stix_nested_ref_relationship["relationship_type"]
+                    in ["operating-system", "sample"]
                 ):
+                    key = (
+                        stix_nested_ref_relationship["relationship_type"]
+                        .replace("obs_", "")
+                        .replace("-", "_")
+                        + "_ref"
+                    )
+                    entity[key] = stix_nested_ref_relationship["to"]["standard_id"]
+
+                else:
                     key = (
                         stix_nested_ref_relationship["relationship_type"]
                         .replace("obs_", "")
                         .replace("-", "_")
                         + "_refs"
                     )
-                    if key in entity:
+                    if key in entity and isinstance(entity[key], list):
```

Comment:
the isinstance is not very pretty. camelCase maybe ?

Reply:
isinstance is a built-in Python function